cli/internal/cmd/iamcreate.go

```diff
@@ -235,10 +233,16 @@ func (c *iamCreator) create(ctx context.Context) error {
 		}
 	}
 
-	c.spinner.Start("Creating", false)
-
-	conf := createConfig(c.provider)
+	var conf config.Config
+	if flags.updateConfig {
+		c.log.Debugf("Parsing config %s", flags.configPath)
```

Comment:
iam create should be possible without parsing an existing config, see
https://docs.edgeless.systems/constellation/getting-started/first-steps